apps/docs/src/examples/renderers/CSS2DRenderer/App.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+  import { Canvas } from '@threlte/core'
+  import Scene from './Scene.svelte'
+</script>
+
+<div id="css-renderer-target" class="absolute top-0 left-0" style="pointer-events: none;" />
```

Comment:
No Tailwind in examples: please move this CSS - including the pointer events - to a `<style>` block. We can use the `#css-renderer-target` id as a selector.